docs/source/quickstart.rst

```diff
@@ -3,6 +3,219 @@
+            log_metric("accuracy", random() + 0.2)
+
+            # Log an artifact (output file)
+            if not os.path.exists("outputs"):
+                os.makedirs("outputs")
+            with open("outputs/test.txt", "w") as f:
+                f.write("hello world!")
+            log_artifacts("outputs")
+
+How do I view my MLFlow runs and experiments?
+---------------------------------------------
+
+Once you've run your code, you may view the results with MLFlow's tracking UI. To start the UI, run:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow ui
+
+And then navigate to http://localhost:5000 in your browser. You will see a page similar to:
+
+.. image:: _static/images/quickstart-ui-screenshot.png
+    :width: 800px
+    :align: center
+
+You are in the **Default** experiment, which now contains the tracking data for your run. You can view the parameters, metrics, and artifacts of your run by clicking on the run name. 
+
+How do I share my MLFlow runs and experiments?
+----------------------------------------------
+
+By default, MLFlow stores tracking data and artifacts in a ``./mlruns`` subdirectory of where you ran the code. You can change this behavior by setting the ``MLFLOW_TRACKING_URI`` environment variable to a different location. For instance, you can set it to a shared filesystem, a SQLAlchemy-compatible database, a tracking server, or a Databricks workspace.
+
+You can run a tracking server on a network-accessible server by running:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow server
+
+On your development machine, set the ``MLFLOW_TRACKING_URI`` environment variable to the URL of that server. For example, if your tracking server is listening on the default port of **5000** on the machine with IP address **192.168.0.1**, you can set the ``MLFLOW_TRACKING_URI`` environment variable to:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        export MLFLOW_TRACKING_URI=http://192.168.0.1:5000
+
+Now, when you run your code, it will send tracking data to the tracking server. To view the results, run the tracking UI on your local machine and point it to the tracking server:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow ui --backend-store-uri http://192.168.0.1:5000
+
+..
+
+    CONTENT REVIEW: Maybe that's all we need for *this* page? Short and sweet, you've: 
+
+    - installed MLFlow
+    - used `autolog` and explicit logging
+    - run the tracking UI
+    - run the tracking server
+
+    Maybe we end there and "Next Steps" it? 
+
+    If not:
+
+How do I use MLFlow with a Databricks workspace?
+------------------------------------------------
+
+tk 
+
+
+How do I store a model in MLFlow?
+---------------------------------
+
+tk 
+
+
+How do I run a model artifact from a specific MLFlow run?
+---------------------------------------------------------
+
+tk
+
+How do I do a hyperparameter sweep with MLFlow Tracking?
```

Comment:
Can probably leave this out for now, but it would be great to call out Projects and Recipes briefly at the end of this workflow. "For more information about creating reproducible model training workflows, check out MLflow Projects.", "For prepackaged training and deployment solutions for classification and regression modeling, check out MLflow Recipes"

Reply:
Removed this H2.